infrastructure/main.tf

```diff
@@ -55,6 +55,7 @@ module "case-management-web" {
   website_local_cache_sizeinmb = 500
   capacity = "${var.capacity}"
   appinsights_instrumentation_key = "${var.appinsights_instrumentation_key}"
+  web_pages_appinsights_instrumentation_key = "${var.web_pages_appinsights_instrumentation_key}"
```

Comment:
this isn't a thing, the above key will be used

Reply:
I wanted to have an express separation between the key for the AI resource of the BE service and the key for that of the FE pages. These keys may be different indeed (as suggested by MS as well).

Reply:
pass it through as an app setting then not as a module parameter

Reply:
MS also suggests one AppInsights across all apps.  I'd prefer 1 tbh as its a single place for all info.  Since this is new lets just use the one we have, we can always separate it out if it proves to not work for us.  As it stands this code won't actually tie into any AppInsights instance that we have.

Reply:
Dwayne, our 1 App Insights resource is collecting everything from Azure-based applications. If we use the inst key of that one for Web Page AI integration, then the known and accepted vulnerabilities will have been accepted to impact the AI for all others. For details, please see:
https://tools.hmcts.net/confluence/display/RCCD/Application+Insights+for+Angular+Clients+on+Browsers
and
https://docs.microsoft.com/en-us/azure/azure-monitor/app/troubleshoot-faq#my-instrumentation-key-is-visible-in-my-web-page-source

On the second link, as you'll see, using a separate AI resource dedicated to web pages is also a mentioned in MS's Q&A page. If we expose the inst key of all-in-one AI resource to public, that may make security approval of this change a lot more difficult.

Reply:
They don't say that you should do it though.
```
This is common practice in monitoring solutions.
It can't be used to steal your data.
It could be used to skew your data or trigger alerts.
We have not heard that any customer has had such problems.
```

I think this is inventing a solution when we don't need it?